### DISCUSSION TITLE
Unable to build after updating to version v3.14.0.4118

### DISCUSSION THREAD

**Discussion by mjanulaitis** [wixtoolset/issues#6524] [Questions]
* Which version of WiX are you building with?
> v3.14.0.4118

* Which version of Visual Studio are you building with (if any)?
> 16.9.3

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?
> NA

* Which version of .NET are you building with?
> 4.7.2

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?
> This is a build issue

* Describe the problem and the steps to reproduce it.
* After upgrading to the latest version I get the following errors when building.  Please note the merge module in question has not changed in over 6 months.

2>light.exe(0,0): error LGHT0204: ICEM10: The property 'ProductCode' is not allowed in a Merge Module
2>C:\Users\mike\AppData\Local\Temp\pg1e2w52\pg1e2w52\ConsoleMergeModule.msm(0,0): error LGHT0204: ICE03: Table: Component Column: _IceM05Mark Missing specifications in _Validation Table (or Old Database)
2>C:\Users\mike\AppData\Local\Temp\pg1e2w52\pg1e2w52\ConsoleMergeModule.msm(0,0): error LGHT0204: ICE03: Table: File Column: _ICEM07CAB Missing specifications in _Validation Table (or Old Database)
2>light.exe(0,0): error LGHT0217: Error executing ICE action 'ICE103'. The most common cause of this kind of ICE failure is an incorrectly registered scripting engine. See http://wixtoolset.org/documentation/error217/ for details and how to solve this problem. The following string format was not expected by the external UI message logger: "There is a problem with this Windows Installer package. A DLL required for this install to complete could not be run. Contact your support personnel or package vendor.  ".

* Describe the behavior you expected and how it differed from the actual behavior.
> I expected the build to complete successfully.

**Comment by barnson** [comment]
Please attach diagnostic-level MSBuild log.

**Comment by mjanulaitis** [comment]
Here you go...

[diagnostics.txt](https://github.com/wixtoolset/issues/files/6853205/diagnostics.txt)

I search all the code and the referenced string is no where to be found:

The property 'ProductCode' is not allowed in a Merge Module

**Comment by barnson** [comment]
This doesn't appear to be a bug. You might take a look at the merge module in Orca.

We moved this issue to the [experimental GitHub discussions area][gh].
You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix

**Comment by robmen** [comment]
Can you share the SharedMergeModule.wxs code and the SharedMergeModule.wixobj file? Either something is wrong in your .wxs code or a very, very strange bug slipped into WiX v3.14 that should show up in SharedMergeModule.wixobj.

**Reply by mjanulaitis** [reply]
Here you go...And thanks for taking a look.

[SharedMergeModule.zip](https://github.com/wixtoolset/issues/files/6932044/SharedMergeModule.zip)

**Reply by mjanulaitis** [reply]
Hello @robmen by any chance have you been able to look at the source code I published?  I would really like to build against the tip as the old version won't install .Net 4.7.2 anymore :(

**Reply by robmen** [reply]
No. It wasn't immediately obvious. I've been pretty burned out for the last couple of months. I've kept up with my customer's requests but haven't felt like digging deeper into random bugs for free.

**Reply by mjanulaitis** [reply]
@robmen I will be creating a new major upgrade this month and will start with an empty project then add on.  Maybe I will see the issue during this processes.  I will let you know what I find out.  Thanks.

**Reply by barnson** [reply]
https://github.com/wixtoolset/issues/issues/6567
